warp10/src/main/java/io/warp10/script/binary/MACROBINARYOP.java

```diff
@@ -0,0 +1,125 @@
+package io.warp10.script.binary;
+
+import io.warp10.continuum.gts.GTSHelper;
+import io.warp10.continuum.gts.GTSOpsHelper;
+import io.warp10.continuum.gts.GeoTimeSerie;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.formatted.FormattedWarpScriptFunction;
+
+import java.util.Map;
+
+public class MACROBINARYOP extends FormattedWarpScriptFunction {
```

Comment:
Given this function applies the macro instead of producing a wrapper around it (in the way MACROREDUCER, MACROMAPPER do), the name should be changed to something else.